source/VisTrace.cpp

```diff
@@ -218,6 +218,48 @@ LUA_FUNCTION(RT_SetPixel)
 	return 0;
 }
 
+LUA_FUNCTION(RT_Load)
+{
+	LUA->CheckType(1, RenderTarget::id);
+	LUA->CheckType(2, Type::String);
+	// 3 and 4 are generateMips and scaleToRT respectively, they are both optional, but generateMips is true by default and cannot use the shorthand method for optional booleans.
+
+	const char* filepath = LUA->GetString(2);
+	bool generateMips = true;
```

Comment:
Should not be generating MIPs by default, it's more performance intensive and won't be used for 99% of applications